From 4bfb4a0e3d55e57c59a9ff1e7defe426d9d7df93 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Fri, 13 Sep 2019 00:27:56 +0200
Subject: [PATCH 100/152] cleanup: Fix wrong indentation

Some more places where the indentation doesn't comply with either
the old or new style. They slipped through because the legacy eslint
configuration accounts for some patterns by plainly ignoring certain
nodes. We'll address that later, first fix up the indentation errors.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/725
---
 js/ui/appDisplay.js                | 10 ++++++----
 js/ui/components/autorunManager.js |  6 +++---
 js/ui/kbdA11yDialog.js             | 22 +++++++++++-----------
 3 files changed, 20 insertions(+), 18 deletions(-)

diff --git a/js/ui/appDisplay.js b/js/ui/appDisplay.js
index a9c26ab97..a718fd428 100644
--- a/js/ui/appDisplay.js
+++ b/js/ui/appDisplay.js
@@ -1884,15 +1884,17 @@ var AppFolderPopup = class AppFolderPopup {
             direction = St.DirectionType.TAB_FORWARD;
             break;
         case Clutter.Right:
-            direction = isLtr ? St.DirectionType.TAB_FORWARD :
-                                    St.DirectionType.TAB_BACKWARD;
+            direction = isLtr
+                ? St.DirectionType.TAB_FORWARD
+                : St.DirectionType.TAB_BACKWARD;
             break;
         case Clutter.Up:
             direction = St.DirectionType.TAB_BACKWARD;
             break;
         case Clutter.Left:
-            direction = isLtr ? St.DirectionType.TAB_BACKWARD :
-                                    St.DirectionType.TAB_FORWARD;
+            direction = isLtr
+                ? St.DirectionType.TAB_BACKWARD
+                : St.DirectionType.TAB_FORWARD;
             break;
         default:
             return Clutter.EVENT_PROPAGATE;
diff --git a/js/ui/components/autorunManager.js b/js/ui/components/autorunManager.js
index 013521baf..cb0c6b6a3 100644
--- a/js/ui/components/autorunManager.js
+++ b/js/ui/components/autorunManager.js
@@ -117,9 +117,9 @@ var ContentTypeDiscoverer = class {
 
             let hotplugSniffer = new HotplugSniffer();
             hotplugSniffer.SniffURIRemote(root.get_uri(),
-                 ([contentTypes]) => {
-                     this._emitCallback(mount, contentTypes);
-                 });
+                ([contentTypes]) => {
+                    this._emitCallback(mount, contentTypes);
+                });
         }
     }
 
diff --git a/js/ui/kbdA11yDialog.js b/js/ui/kbdA11yDialog.js
index ba3576599..f5ffccce7 100644
--- a/js/ui/kbdA11yDialog.js
+++ b/js/ui/kbdA11yDialog.js
@@ -28,23 +28,23 @@ class KbdA11yDialog extends GObject.Object {
         if (whatChanged & Clutter.KeyboardA11yFlags.SLOW_KEYS_ENABLED) {
             key = KEY_SLOW_KEYS_ENABLED;
             enabled = (newFlags & Clutter.KeyboardA11yFlags.SLOW_KEYS_ENABLED) > 0;
-            title = enabled ?
-                    _("Slow Keys Turned On") :
-                    _("Slow Keys Turned Off");
+            title = enabled
+                ? _("Slow Keys Turned On")
+                : _("Slow Keys Turned Off");
             body = _("You just held down the Shift key for 8 seconds. This is the shortcut " +
                      "for the Slow Keys feature, which affects the way your keyboard works.");
 
         } else  if (whatChanged & Clutter.KeyboardA11yFlags.STICKY_KEYS_ENABLED) {
             key = KEY_STICKY_KEYS_ENABLED;
             enabled = (newFlags & Clutter.KeyboardA11yFlags.STICKY_KEYS_ENABLED) > 0;
-            title = enabled ?
-                    _("Sticky Keys Turned On") :
-                    _("Sticky Keys Turned Off");
-            body = enabled ?
-                   _("You just pressed the Shift key 5 times in a row. This is the shortcut " +
-                     "for the Sticky Keys feature, which affects the way your keyboard works.") :
-                   _("You just pressed two keys at once, or pressed the Shift key 5 times in a row. " +
-                     "This turns off the Sticky Keys feature, which affects the way your keyboard works.");
+            title = enabled
+                ? _("Sticky Keys Turned On")
+                : _("Sticky Keys Turned Off");
+            body = enabled
+                ? _("You just pressed the Shift key 5 times in a row. This is the shortcut " +
+                  "for the Sticky Keys feature, which affects the way your keyboard works.")
+                : _("You just pressed two keys at once, or pressed the Shift key 5 times in a row. " +
+                  "This turns off the Sticky Keys feature, which affects the way your keyboard works.");
         } else {
             return;
         }
-- 
2.23.0

